tools/stress/src/org/apache/cassandra/stress/Stress.java

```diff
@@ -88,6 +94,14 @@ private static int run(String[] arguments)
                 return 1;
             }
 
+            if (settings.log.printThreadDumpOnShutdown) {
+                Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+                    System.out.println("Shutdown thread dump:");
```

Comment:
It should not print thread dump when it terminates in a regular way, only when it is terminated by signal

Reply:
Removed